src/rgw/rgw_rados.cc

```diff
@@ -6196,122 +6189,6 @@ int RGWRados::Object::Read::read(int64_t ofs, int64_t end, bufferlist& bl, optio
   return bl.length();
 }
 
-
-/*
-
-struct get_obj_data : public RefCountedObject{
```

Comment:
um, maybe we should just remove this from prior commits, rather than putting it in an taking it out later